molgenis-web/src/main/java/org/molgenis/web/bootstrap/PluginPopulator.java

```diff
@@ -31,28 +30,22 @@ public PluginPopulator(DataService dataService, PluginFactory pluginFactory)
 
 	public void populate(ApplicationContext ctx)
 	{
-		Map<String, Plugin> newOrChangedPluginMap = getPlugins(ctx);
-		List<Plugin> deletedPlugins = new ArrayList<>();
+		Map<String, Plugin> allPlugins = getPlugins(ctx); // Plugins from context
```

Comment:
Are these changes related to the description update? If this a bug fix? In case of a bug fix create an issue and use a separate PR.

Reply:
Yeah this is a fix where the plugins from the context always overwrite the plugins in the database at startup.

In practice this means that the plugin description (which is filled in at runtime) is overwritten with NULL by the context when the server is rebooted.

Il make a separate PR